filestack/Filelink.php

```diff
@@ -38,6 +49,9 @@ public function __construct($handle, $api_key = '', $security = null,
         }
 
         $this->http_client = $http_client; // CommonMixin
+
+        $cname = filter_var($cname, FILTER_VALIDATE_DOMAIN, FILTER_FLAG_HOSTNAME);
```

Comment:
Looks like `FILTER_VALIDATE_DOMAIN` is not PHP 5.x compatible. Not sure how important that compatibility is for this library.